nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -140,10 +158,25 @@ global_costmap:
           max_obstacle_height: 2.0
           clearing: True
           marking: True
+          data_type: "LaserScan"
+      voxel_layer:
+        enabled: True
+        publish_voxel_map: True
+        origin_z: 0.0
+        z_resolution: 0.2
+        z_voxels: 10
+        max_obstacle_height: 2.0
+        mark_threshold: 0
+        pointcloud:
+          topic: /intel_realsense_r200_depth/points
+          max_obstacle_height: 2.0
+          clearing: True
+          marking: True
+          data_type: "PointCloud2"
       static_layer:
         map_subscribe_transient_local: True
       always_send_full_costmap: True
-      observation_sources: scan
+      observation_sources: scan pointcloud
```

Comment:
These observation sources should be in each individual layer’s namespaces. This is what allows you to have multiple sensors processed in the same layer

Reply:
Is this being properly processed as is? Or is it the defaults masking it? That’s a problem

Reply:
Observation sources are now in individual layer's namespaces.